src/Console/ModelsCommand.php

```diff
@@ -484,7 +484,7 @@ public function getPropertiesFromTable($model)
 
         $database = null;
         if (strpos($table, '.')) {
-            [$database, $table] = explode('.', $table);
+            [$database, $table] = explode('.', $table, 2);
```

Comment:
So, my nemesis (the code, not you) is appearing again.

I had a fix for this some years ago, but it failed properly work in all environments -> https://github.com/barryvdh/laravel-ide-helper/pull/974/files

Although my code looks, err..., weird, it actually does the same 😬

Yours:
```
>>> explode('.', 'foo.bar', 2);
=> [
     "foo",
     "bar",
   ]
>>> explode('.', 'foo.bar.baz', 2);
=> [
     "foo",
     "bar.baz",
   ]
>>>
```
Mine:
```
>>> $table = 'foo.bar'; strtok($table, '.');
=> "foo"
>>> strtok('');
=> "bar"
>>>
>>> $table = 'foo.bar.baz'; strtok($table, '.');
=> "foo"
>>> strtok('');
=> "bar.baz"
```

Obviously your solution is smarter 💪🏼 

But in the end, it's the same outcome and [then we had the issue reported it does not work with MSSQL](https://github.com/barryvdh/laravel-ide-helper/pull/974#issuecomment-659565923) (which I don't have to test), so I had to drop it.

Reply:
hmm, wasn't aware of the MSSQL issue 😕 .  I took another stab at this and I think it's not necessary to even attempt to compute the `$database` and `$table` from exploding but rely on the connection information altogether.  I just pushed what I think might be a better approach to the PR.

I don't have access to MSSQL to test this.  This seems to work fine on pgsql/mysql/sqlite.

Reply:
I can confirm it works great on mssql!

Reply:
@mfn Sorry to bother, but anything else you want me to do here?

Reply:
@SimplyCorey I don't think so, but I haven't had time to come back to the issue!